Gemfile

```diff
@@ -12,7 +12,7 @@ ruby '3.1.3'
 gem 'metadata_presenter', '3.3.31'
 
 gem 'activerecord-session_store'
-gem 'administrate'
+gem 'administrate', '~> 0.20.1'
```

Comment:
I've locked the gem as soon they will release v1.0.0 with potentially breaking changes.